From 56c732c9961836e870b7acecc76fd99b68b835a1 Mon Sep 17 00:00:00 2001
From: Son Lam <son.lam.ym@renesas.com>
Date: Sun, 14 Jun 2020 13:41:13 +0700
Subject: [PATCH] ckermit: fix error FILE {aka struct _IO_FILE } has no member
 named _cnt

This commit fixes FILE {aka struct _IO_FILE } has no member named _cnt
due to removal of libio.h of glibc version 2.28

Signed-off-by: Son Lam <son.lam.ym@renesas.com>
---
 ckucmd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ckucmd.c b/ckucmd.c
index 274dc2d..2592cd2 100644
--- a/ckucmd.c
+++ b/ckucmd.c
@@ -7370,7 +7370,7 @@ cmdconchk() {
 
 /* Here we must look inside the stdin buffer - highly platform dependent */
 
-#ifdef _IO_file_flags			/* Linux */
+#ifdef __FILE_defined			/* Linux */
     x = (int) ((stdin->_IO_read_end) - (stdin->_IO_read_ptr));
     debug(F101,"cmdconchk _IO_file_flags","",x);
 #else  /* _IO_file_flags */
-- 
2.7.4

